integration-tests/common-test-code/src/test/java/com/vaadin/flow/quarkus/it/SmokeTest.java

```diff
@@ -26,7 +25,8 @@ public void smokeTest_clickButton() {
 
         button.click();
 
-        // As we don't have CDI functionality for Route annotations we check that we are in
+        // As we don't have CDI functionality for Route annotations we check
+        // that we are in
```

Comment:
This comment is actually wrong and should be removed.

Reply:
done. Even though it's out of scope of the PR and formatting is done automatically.